### PR TITLE
Obsoleted DIC, SYL, HVY - Update facilities.json

### PR DESCRIPTION
**Pull request by japierce1028**
Pointing to TMF

### CODE REVIEW

**Comment by japierce1028** [comment]
I know there is an error, but I don't see it. HELP!

File: src/main/data/facilities.json [hunk 1]
```diff
@@ -1749,7 +1749,11 @@
   },
   {
     "code": "DIC",
-    "description": "DIGIMAGE CINEMA"
+    "description": "DIGIMAGE CINEMA",
+        "obsolete": true,
+    "obsoletedBy": [
+      "TMF"
+    ],
```
Comment (palemieux) [inline src/main/data/facilities.json:1756]:
```suggestion
    ]
```

File: src/main/data/facilities.json [hunk 2]
```diff
@@ -3009,7 +3013,11 @@
   },
   {
     "code": "HVY",
-    "description": "HIVENTY"
+    "description": "HIVENTY",
+        "obsolete": true,
+    "obsoletedBy": [
+      "TMF"
+    ],
```
Comment (palemieux) [inline src/main/data/facilities.json:3020]:
```suggestion
    ]
```

File: src/main/data/facilities.json [hunk 3]
```diff
@@ -5737,7 +5745,11 @@
   },
   {
     "code": "SYL",
-    "description": "SYLICONE"
+    "description": "SYLICONE",
+        "obsolete": true,
+    "obsoletedBy": [
+      "TMF"
+    ],
```
Comment (palemieux) [inline src/main/data/facilities.json:5752]:
```suggestion
    ]
```

Cannot have a hanging comma.

**Comment by palemieux** [comment]
> I know there is an error, but I don't see it. HELP!

Hanging commas are not permitted in JSON. I fixed two of them. Wanna fix the third one?

**Comment by japierce1028** [comment]
At least it is a different error now. I still don't see it, but that's what I do...

File: src/main/data/facilities.json [hunk 4]
```diff
@@ -1749,7 +1749,11 @@
   },
   {
     "code": "DIC",
-    "description": "DIGIMAGE CINEMA"
+    "description": "DIGIMAGE CINEMA",
+        "obsolete": true,
```
Comment (palemieux) [inline src/main/data/facilities.json:1753]:
```suggestion
  "obsolete": true,
```

File: src/main/data/facilities.json [hunk 5]
```diff
@@ -3014,7 +3018,11 @@
   },
   {
     "code": "HVY",
-    "description": "HIVENTY"
+    "description": "HIVENTY",
+        "obsolete": true,
```
Comment (palemieux) [inline src/main/data/facilities.json:3022]:
```suggestion
  "obsolete": true,
```

File: src/main/data/facilities.json [hunk 6]
```diff
@@ -5742,7 +5750,11 @@
   },
   {
     "code": "SYL",
-    "description": "SYLICONE"
+    "description": "SYLICONE",
+        "obsolete": true,
```
Comment (palemieux) [inline src/main/data/facilities.json:5754]:
```suggestion
  "obsolete": true,
```

**Review by japierce1028 — COMMENTED** [review]
Damn, just another comma?

**Review by palemieux — APPROVED** [review]

**Pull request merged**